static/js/components/email/lib_test.js

```diff
@@ -187,8 +188,9 @@ describe('Specific email config', () => {
     it('shouldnt use the sendMail email action, if the email config specifies differently', () => {
       let automaticEmailState = _.clone(filledOutEmailState);
       automaticEmailState.inputs.id = 1;
-      helper.fetchJSONWithCSRFStub.withArgs('/api/v0/mail/automatic_email/1/').
-        returns(Promise.resolve());
+      fetchMock.mock('/api/v0/mail/automatic_email/1/', () => (
```

Comment:
Is this change related to this PR? It looks like it came from the automatic emails PR